### PR TITLE
Fixes accessibility issues in edition switcher, meganav titles are no…

### PR DESCRIPTION
**Pull request by AlbertoElias**
…t buttons anymore in the desktop layout, edition dropdown does not get cut off if search is open. Also simplifies markup and leaves it in an almost final state, we might add o-expander later

cc @triblondon

### CODE REVIEW

File: demos/src/main.mustache [hunk 1]
```diff
@@ -3,40 +3,28 @@
...
 				</div>
 
-				<nav class="o-header__tools o-header__tools--masthead" role="navigation" aria-label="Useful tools">
+				<nav class="o-header__tools" role="navigation" aria-label="Useful tools">
 					<ul class="o-header__tools-items">
-						<li class="o-header__tools-item o-header__tools-item--account o-header__tools-item--logged-in">
-							<a class="o-header__tools-link" href="/myft">
-								<span><i>my</i>FT</span>
-							</a>
+						<li class="o-header__tools-item o-header__tools-item--logged-in">
```
Comment (triblondon) [inline demos/src/main.mustache]: thought we were removing the logged-in classes

File: demos/src/main.mustache [hunk 2]
```diff
@@ -44,62 +32,45 @@
...
-											<span class="o-header__nav-link o-header__nav-link--toggle">
-												<span class="o-header__nav-link--open">Menu</span>
-												<span class="o-header__nav-link--close">Close</span>
-											</span>
+										<button class="o-header__nav-link o-header__nav-link--toggle" data-o-header-togglable data-o-header-togglable-nav>
+											<span class="o-header__meganav-toggle o-header__meganav-toggle--open">Menu</span>
+											<span class="o-header__meganav-toggle o-header__meganav-toggle--close">Close</span>
 										</button>
 
-										<ol class="o-header__meganav o-expander__content">
```
Comment (triblondon) [inline demos/src/main.mustache]: remind me why we removed expander here?

**Comment by triblondon** [comment]
This looks really good. I need a walkthough of the CSS though.  Alice is passing on review so if we could just do that then I'm happy.

File: demos/src/main.mustache (hunk 2 above)
Comment (AlbertoElias) [inline demos/src/main.mustache]: Because it's not being used

File: demos/src/main.mustache (hunk 1 above)
Comment (AlbertoElias) [inline demos/src/main.mustache]: Missed that one!

**Comment by AlbertoElias** [comment]
Probably better to review CSS in next beta as in that one it's what I'm going to focus on

**Comment by triblondon** [comment]
In spec meeting we agreed that this PR only needs markup review. Next beta will focus on CSS review

**Pull request merged**